pyproject.toml

```diff
@@ -1,12 +1,50 @@
-[tool.coverage.run]
-omit = [
-    "viadot/flows/azure_sql_transform_dbt.py",
-    "viadot/flows/supermetrics_to_adls.py",
-    "viadot/flows/supermetrics_to_azure_sql_v2.py"
-    ]
-
-[tool.pytest.ini_options]
-log_format = "%(asctime)s %(levelname)s %(message)s"
-log_date_format = "%Y-%m-%d %H:%M:%S"
-log_cli=true
-log_level="WARNING"
\ No newline at end of file
+tool = { rye = { dev-dependencies = [
+    "black==22.6.0",
+    "pytest==8.1.1",
+    "pytest-cov==3.0.0",
+    "mkdocs-autorefs==0.4.1",
+    "mkdocs-material-extensions==1.0.3",
+    "mkdocs-material==8.4.3",
+    "mkdocs==1.3.1",
+    "mkdocstrings==0.19.0",
+    "Faker==13.12.1",
+    "python-dotenv>=1.0.1",
+] } }
```

Comment:
The [project] key should be at the top.

Reply:
[⬇️ Downgraded python version to >=3.9](https://github.com/dyvenia/viadot/pull/878/commits/f5a296d504c5a931c51d586bfc29460f5a2f49d7)